src/frontend/src/styles/themes.scss

```diff
@@ -0,0 +1,33 @@
+@import 'colors';
+
+// Default theme
+$theme-map-light: (
+    $sms-color-subtle-blue: #7ab1dd,
+    $sms-color-subtle-green: #80ad8b,
+
+    $sms-color-background-brand: #e4671b,
+    $sms-color-background-brand-alt: #cd4058,
+
+    $sms-color-neutral-background: #cacaca,
+    $sms-color-neutral-background-active: #acacac,
+    $sms-color-background: #f4f4f4,
+    $sms-colour-danger-zone: #e10000,
+
+    // bind CDS mappings to our branding
+    $cds-overrides--cds-border-interactive: var($sms-color-background-brand),
+    $cds-overrides--cds-button-primary-active: var($sms-color-background-brand-alt),
+    $cds-overrides--cds-button-primary-hover: var($sms-color-background-brand-alt),
+    $cds-overrides--cds-button-primary: var($sms-color-background-brand),
+    $cds-overrides--cds-focus: var($sms-color-background-brand)
+);
```

Comment:
@Ryan-Cleminson This allows plugging in theme variables with scss vars. 

The `colors.scss` file declares the token names (eg. `$sms-color-my-color`) or overriding CDS tokens (eg.`--cds-button-primary`).

This section instantiates those declared tokens with a HEX colour value and injects it into the DOM root.

Reply:
^ the outcome of this is,

* developers can use intellisense for scss vars
* the stylesheet gets vanilla CSS variables